app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/state/AdditionalOptionMenuState.kt

```diff
@@ -40,6 +39,7 @@ enum class AdditionalOptionSubMenuState {
 
 enum class AdditionalOptionSelectItem {
     RichTextEditing,
+    // it's only used to show keyboard after self deleting bottom sheet collapses
```

Comment:
question: should it always work this way? What if the user wants to send a self deleting image message? Then opening a keyboard always after choosing the self deleting option complicates it a bit for the user as he needs to re-open the additional options menu again 🤔

Reply:
I think we can improve that. But currently I don't want complicate code for this small UX behaviour